themes/gatsby-theme-conference/gatsby-node.js

```diff
@@ -0,0 +1,47 @@
+const fs = require('fs-extra')
+const path = require('path')
+const mkdirp = require('mkdirp')
+const Debug = require('debug')
+const pkg = require('./package.json')
+
+const debug = Debug(pkg.name)
+
+const Landing = require.resolve('./src/templates/index')
+const Speakers = require.resolve('./src/templates/speakers')
+const Schedule = require.resolve('./src/templates/schedule')
+
+exports.onPreBootstrap = ({ store }) => {
+  const { program } = store.getState()
+
+  const dir = path.join(program.directory, `src/data`)
+
+  debug(`Initializing ${dir} directory`)
+  mkdirp.sync(dir)
+
+  // todo: find a better way to bootstrap site data
```

Comment:
What's the best way to handle this sort of thing? cc @johno

Reply:
That's basically what I've been doing. Not sure if there's really a better way tbh.

Reply:
My thought is that themes could include example data and we could let people choose whether or not they want to create the site or not with that data.